sunpy/io/cdf.py

```diff
@@ -182,7 +182,11 @@ def read_cdf(fname):
                 '1/(cm2 Sr sec MeV/nucleon)': 1 / (u.cm**2 * u.sr * u.s * u.MeV),
                 '1/(cm**2-s-sr-MeV)': 1 / (u.cm**2 * u.s * u.sr * u.MeV),
                 '1/(cm**2-s-sr-MeV/nuc.)': 1 / (u.cm**2 * u.s * u.sr * u.MeV),
+                '(cm^2 s sr MeV/n)^-1': 1 / (u.cm**2 * u.s * u.sr * u.MeV),
+                'cm!E-2!Nsr!E-1!Nsec!E-1!N(MeV/nuc)!E-1!N': 1 / (u.cm**2 * u.s * u.sr * u.MeV),
```

Comment:
:eyes: :scream_cat: :sweat:

Reply:
I double-checked the [PSP/IS⊙IS Energetic Particle Data - User Guide](https://spp-isois.sr.unh.edu/data_public/ISOIS_Data_Glossary.pdf) because that notation (which I'm not used to) scared me 😅